newIDE/app/src/ProjectManager/index.js

```diff
@@ -344,7 +346,17 @@ export default class ProjectManager extends React.Component<Props, State> {
     newLayout.setName(newName); // Unserialization has overwritten the name.
     newLayout.updateBehaviorsSharedData(project);
 
-    this.forceUpdate();
+    this._onModifiedObject();
+  };
+
+  _duplicateLayout = (layout: gdLayout, index: number) => {
+    this._copyLayout(layout);
+    this._pasteLayout(index);
+  };
+
+  _duplicateLayout = (layout: gdLayout, index: number) => {
+    this._copyLayout(layout);
+    this._pasteLayout(index);
   };
```

Comment:
_duplicateLayout seems... duplicated ;) 
Check also the other _duplicateXXX in this file.